src/site/content/en/blog/native-file-system/index.md

```diff
@@ -0,0 +1,465 @@
+---
+title: "The Native File System API: Simplifying access to local files"
+subhead: The new Native File System API enables developers to build powerful web apps that interact with files on the user's local device, like IDEs, photo and video editors, text editors, and more. After a user grants a web app access, this API allows web apps to read or save changes directly to files and folders on the user's device.
```

Comment:
For all Fugu posts, the subhead should probably just be a sentence or two. The subhead appears on the /blog page and currently these are much longer than the other ones.

![image](https://user-images.githubusercontent.com/1066253/66506934-1c6c8480-ea83-11e9-857c-bd680368b96c.png)

Also, the subhead appears at the top of the article so its text should be different from text in the article itself.

<img width="940" alt="Screen Shot 2019-10-09 at 10 55 13 AM" src="https://user-images.githubusercontent.com/1066253/66507148-82590c00-ea83-11e9-8836-2f147e630ef2.png">